index.html

```diff
@@ -10,21 +10,21 @@
     <h2>Jogo da Velha</h2>
     <table>
         <tr> 
-            <td><div id='pos6' onclick="clic(6)">7</div></td>
-            <td><div id='pos7' onclick="clic(7)">8</div></td>
-            <td><div id='pos8' onclick="clic(8)">9</div></td>
+            <td><div class='cell' id='pos00' onclick="clic(0,0)">0,0</div></td>
```

Comment:
It is a good practice to always use double quotes (`"`) for HTML properties rather than single quotes (`'`).

```html
<div class="cell" id="pos00" onclick="clic(0, 0)">0,0</div>
```